### PR TITLE
Add unlimited universal transformer script

### PR DESCRIPTION
**Pull request by serdardoesml**
Added a scaled up version with 2b total parameters repeating without schedule to depth 48, with a lighter version of mtp using only a mtp lm_head, no mtp block. Also reduced batch size and adjusted lr and wd accordingly. Trying to get it to a point where together with ensembling and chain distillation it could be competitive in the unlimited track.

Unlimited training summary:
Total training time: 814.01m
Final train loss: 3.933960
Min val BPB: 1.040571
Min val Loss: 3.201859
Total wall time: 51953.06s (865.88m)

Note: These numbers are without the document level shuffling change that was applied when merging.

### CODE REVIEW

**Comment by akshayvegesna** [comment]
Thanks for your work here. Just one nit: can you name the unlimited script train_unlimited -- python files are generally not named with a dash.

**Comment by serdardoesml** [comment]
> Thanks for your work here. Just one nit: can you name the unlimited script train_unlimited -- python files are generally not named with a dash.

Ah my bad, done.
